src/main/java/lotto/domain/Rank.java

```diff
@@ -1,43 +1,47 @@
 package lotto.domain;
 
-import java.util.Arrays;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 public enum Rank {
-    FIRST(6, new Money(2_000_000_000L), "6개 일치 (2000000000원)- %d개"),
-    SECOND(5, new Money(30_000_000L), "5개 일치, 보너스 볼 일치(30000000원)- %d개"),
-    THIRD(5, new Money(1_500_000L), "5개 일치 (1500000원)- %d개"),
-    FOURTH(4, new Money(50_000L), "4개 일치 (50000원)- %d개"),
-    FIFTH(3, new Money(5_000L), "3개 일치 (5000원)- %d개"),
-    MISS(0, new Money(0L), "");
+    FIRST(6, new Money(2_000_000_000L)),
+    SECOND(5, new Money(30_000_000L)),
+    THIRD(5, new Money(1_500_000L)),
+    FOURTH(4, new Money(50_000L)),
+    FIFTH(3, new Money(5_000L)),
+    MISS(0, new Money(0L));
 
     private final int count;
     private final Money money;
-    private final String message;
+    private static final Map<Long, Rank> RANK_CACHE;
```

Comment:
```suggestion
    private static final Map<Long, Rank> RANK_CACHE = Arrays.stream(values())
            .collect(Collectors.toMap(Rank::getCount, Function.identity()));
```
이런 식으로 하면 매번 추가할 때마다 static에서 수기로 코드를 추가할 필요가 없어요~